site/app/templates/generic/Popup.twig

```diff
@@ -31,6 +31,7 @@
                                                 onclick="{% block close_click_action %}closePopup('{{ block('popup_id') }}');{% endblock %}"
                                                 class="btn {% block close_button_classes %}btn-default{% endblock %} close-button key_to_click"
                                                 tabindex="0"
+                                                data-testid="form-button-container"
```

Comment:
Is this being used anywhere?

Reply:
NO , i just removed this . Thank you for review . File name also changed from **prev_next.spec.js** to **cyclic_grading_view.spec.js**